tests/integration/results.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2023 Democratized Data Foundation
```

Comment:
praise: Thanks for moving this to a new file, is much tidier, and I think Shahzad had been wanting this for a while :)